desk/app/logs.hoon

```diff
@@ -77,21 +61,17 @@
     =+  !<(=a-log:l vase)
     ?-    -.a-log
         %log
-      ?>  ?=([@ @ ~] sap.bowl)
-      =+  agent=i.t.sap.bowl
-      =^  id-event  logs  (add-log-event agent +.a-log)
-      (send-posthog-event agent id-event +.a-log)
+      =/  =log-item:l  [now.bowl +.a-log]
+      (send-posthog-event sap.bowl log-item)
```

Comment:
Given that the `+send-posthog-event` sample also specifies time and event separately, it'd be fine to not explicitly construct a `$log-item` here.

```suggestion
      (send-posthog-event sap.bowl now.bowl +.a-log)
```